src/views/core/bodies/Single.vue

```diff
@@ -96,6 +96,11 @@
                   <th>Title</th>
                   <td>{{ body.name }}</td>
                 </tr>
+                <tr v-if="body.type != 'antenna' && body.type != 'contact antenna' && body.type != 'contact'">
```

Comment:
`v-if="['antenna', 'contact antenna', 'contact'].includes(body.type)"`

Reply:
Ah, I was trying out checking if the body.type was in an array but that did not really work. I'll change this later today